src/integrations/FacebookPixel/browser.js

```diff
@@ -205,6 +210,10 @@ class FacebookPixel {
             content_ids: contentIds,
             content_type: this.getContentType(rudderElement, contentType),
             contents,
+            content_category: category || "",
+            content_name: contentName,
+            value: this.formatRevenue(properties?.value),
```

Comment:
product list viewed doesn't have value property. Make sure this is added in facebook pixel documentation